composer.json

```diff
@@ -5,17 +5,19 @@
     "license": "MIT",
     "description": "Encrypted symfony entity's by verified and standardized libraries",
     "require": {
-        "php": ">=7.0",
+        "php": "^7.2",
```

Comment:
halite 4.3 requires php 7.2 at least, no point in allowing a lower version here.

and its better to specify the php version with `^` so we don't accidentally install with php 8 without knowing if there are BC breaks.